pkgs/misc/vscode-extensions/rescript/rescript-editor-analysis.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, pkgs }:
+let
+  src = fetchFromGitHub {
+    name = "rescript-vscode";
+    owner = "rescript-lang";
+    repo = "rescript-vscode";
+    rev = "8d0412a72307b220b7f5774e2612760a2d429059";
+    sha256 = "rHQtfuIiEWlSPuZvNpEafsvlXCj2Uv1YRR1IfvKfC2s=";
+  };
+in
+stdenv.mkDerivation {
+  inherit src;
+  name = "rescript-editor-analysis";
+  nativeBuildInputs = with pkgs; [ bash ocaml ];
```

```suggestion
  nativeBuildInputs = [ ocaml ];
```
bash is in the stdenv IIRC.